scripts/murmur.conf

```diff
@@ -1,12 +0,0 @@
-<!DOCTYPE busconfig PUBLIC
```

Comment:
This file is in fact currently still installed in the Arch Linux package, albeit with changed user name.

I'm not entirely sure whether it is still needed. Someone who is more familiar with how you are using/ interacting with dbus should comment on this file.

Potentially it should remain and be adapted to the new name.